src/shared/components/post/post-listing.tsx

```diff
@@ -1153,20 +1153,20 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
             ) : (
               <>
                 <button
-                  className="d-inline-block mr-1 btn btn-link btn-animate text-muted py-0"
+                  className="post-listing__btn post-listing__btn--transfc-confirm d-inline-block mr-1 btn btn-link btn-animate text-muted py-0"
                   aria-label={i18n.t("are_you_sure")}
                 >
                   {i18n.t("are_you_sure")}
                 </button>
                 <button
-                  className="btn btn-link btn-animate text-muted py-0 d-inline-block mr-1"
+                  className="post-listing__btn post-listing__btn--htransfc btn btn-link btn-animate text-muted py-0 d-inline-block mr-1"
                   aria-label={i18n.t("yes")}
                   onClick={linkEvent(this, this.handleTransferCommunity)}
                 >
                   {this.state.transferLoading ? <Spinner /> : i18n.t("yes")}
                 </button>
                 <button
-                  className="btn btn-link btn-animate text-muted py-0 d-inline-block"
+                  className="post-listing__btn post-listing__btn--cshtransfc btn btn-link btn-animate text-muted py-0 d-inline-block"
```

Comment:
`cshtransfc`?

Reply:
Named after existing method contained within -  "handle cancel show confirm transfer community". I guess that would be "hcsctransfc" or "csctransfc".  Can do "--cancel-transfc".